app/uk/gov/hmrc/gform/sharedmodel/VariadicFormData.scala

```diff
@@ -213,8 +216,103 @@ object VariadicFormData {
     // properly classify data to VariadicValues
     val multiValueIds: Set[BaseComponentId] = formModel.allMultiSelectionIds.map(_.baseComponentId)
 
+    val dataGroupBy = data.groupBy(_._1.baseComponentId)
```

Comment:
use of _1 in tuples makes it hard to read. Can we use case expressions instead please?